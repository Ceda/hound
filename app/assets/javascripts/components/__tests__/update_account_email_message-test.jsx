import UpdateAccountEmailMessage from '../update_account_email_message.js';

it('renders appropriately', () => {
  const wrapper = shallow(
    <UpdateAccountEmailMessage
      addressChanged={false}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
