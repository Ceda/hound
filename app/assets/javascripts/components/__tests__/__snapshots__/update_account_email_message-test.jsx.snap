exports[`test renders appropriately 1`] = `
<p
  className="inline-flash inline-flash--error">
  <i
    className="fa fa-exclamation-circle">
    There was a problem updating your email. Please try again.
  </i>
</p>
`;
